plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -62,14 +62,14 @@ class ServiceBreadcrumbs extends React.Component {
   checkBreadcrumbOverflow() {
     if (this.primaryBreadcrumbTextRef != null) {
       const availableWidth = this.getAvailableBreadcrumbWidth();
-      const statusBarWidth = this.getBreadcrumbStatusBarWidth();
+      const ProgressBarWidth = this.getBreadcrumbProgressBarWidth();
 
       if (availableWidth <= 0 && this.state.shouldRenderServiceStatus) {
         // Hide the status bar when the breadcrumbs are wider than the
         // container's width.
         this.setState({ shouldRenderServiceStatus: false });
       } else if (
-        availableWidth > statusBarWidth &&
+        availableWidth > ProgressBarWidth &&
```

Comment:
Sorry for the detail, but better to keep camelCase for consistency.

Reply:
not detail, you are right sir. I was on automatic changing everything and missed this guy is a variable